assets/questions/bestMeetupLocation.js

```diff
@@ -0,0 +1,135 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Best Meeting Location for N People in
+ * a Manhattan-like City.
+ */
+
+globalData.questions['bestMeetupLocation'] = {  // eslint-disable-line dot-notation
+  title: 'Best Meeting Location for N People in a Manhattan-like City',
+  starterCode: {
+    python:
+`def bestMeetupLocation(people):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def findAverageLocation(cls, people):
+        numPeople = len(people)
+        if numPeople == 0:
+            return ""
+        x = 0
+        y = 0
+        for p in people:
+            [dx, dy] = p.split(", ")
+            x += int(dx)
+            y += int(dy)
+        x /= numPeople
+        y /= numPeople
+        return str(x) + ", " + str(y)
+`
+  },
+  tasks: [{
+    instructions: [
+      [
+        'N people live in a city with streets that form a regular 2D grid. ',
+        'They start out at different intersections and want to meet up. ',
+        'They are only able to move along the roads (no crossing blocks diagonally etc.). ',
+        'Find the meeting place that minimizes the total travel distance. ',
```

Comment:
This sentence isn't necessary -- it's essentially restated by the next sentence.